src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -165,7 +158,7 @@ export class CalcitePopover {
 
   @Element() el: HTMLCalcitePopoverElement;
 
-  @State() _referenceElement: HTMLElement = this.getReferenceElement();
+  @State() _referenceElement: HTMLElement;
```

Comment:
Nitpick: To avoid confusion with the prop, maybe this could be renamed to `effectiveReferenceElement` or similar?

Reply:
Yeah I like that.